src/features/api-keys/routes.ts

```diff
@@ -1,4 +1,5 @@
 import { errors } from '@balena/pinejs';
+import { BadRequestError } from '@balena/pinejs/out/sbvr-api/errors';
```

Comment:
We shouldn't import from a module's nested exports.

You have access to `BadRequest` via `errors` and in fact it's already used in a couple of places in this file.